task/secure/src/main.rs

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#![feature(naked_functions)]
+#![no_std]
+#![no_main]
+
+use hypocalls::*;
+#[allow(unused_imports)]
+use userlib::*;
+
+#[link_section = ".tz_table"]
+#[no_mangle]
+#[used]
+static TZ_TABLE: SecureTable = SecureTable {
+    magic: 0xaaaa_bbbb,
```

Comment:
Is this number important?

Reply:
Not particularly. I added the magic number mostly to be an extra check to make sure the table was consistent in the task using the table. Clearly I didn't end up implementing that. I'll see about actually adding that check.